src/test/regress/sql/multi_follower_dml.sql

```diff
@@ -0,0 +1,57 @@
+\c - - - :master_port
+
+CREATE TABLE the_table (a int, b int, z bigserial);
+SELECT create_distributed_table('the_table', 'a');
+
+\c - - - :follower_master_port
+
+-- inserts normally do not work on a standby coordinator
+INSERT INTO the_table (a, b, z) VALUES (1, 2, 2);
+
+-- we can allow DML on a writable standby coordinator
+SET citus.writable_standby_coordinator TO on;
+
+INSERT INTO the_table (a, b, z) VALUES (1, 2, 2);
+SELECT * FROM the_table;
+
+UPDATE the_table SET z = 3 WHERE a = 1;
+SELECT * FROM the_table;
+
+DELETE FROM the_table WHERE a = 1;
+SELECT * FROM the_table;
+
+-- drawing from a sequence is not possible
+INSERT INTO the_table (a, b) VALUES (1, 2);
+
+-- 2PC is not possible
+INSERT INTO the_table (a, b, z) VALUES (2, 3, 4), (5, 6, 7);
+
+-- 1PC is possible
+SET citus.multi_shard_commit_protocol TO '1pc';
+INSERT INTO the_table (a, b, z) VALUES (2, 3, 4), (5, 6, 7);
+SELECT * FROM the_table ORDER BY a;
+
+-- modifying CTEs are possible
+WITH del AS (DELETE FROM the_table RETURNING *)
+SELECT * FROM del ORDER BY a;
+
+-- DDL is not possible
+TRUNCATE the_table;
+ALTER TABLE the_table ADD COLUMN c int;
+
+-- rollback is possible
+BEGIN;
+INSERT INTO the_table (a, b, z) VALUES (1, 2, 2);
+ROLLBACK;
+
+SELECT * FROM the_table;
+
```

Comment:
Maybe also add a test to verify that we cannot write into regular postgres tables.

Reply:
Added